Bugsnag/KSCrash/Source/KSCrash/Recording/BSG_KSCrash.m

```diff
@@ -38,9 +38,10 @@
 #import "BSG_KSLogger.h"
 #import "BugsnagThread.h"
 #import "BSGSerialization.h"
-#import "BugsnagErrorReportSink.h"
+#import "Bugsnag.h"
 #import "BugsnagCollections.h"
 #import "BSG_KSCrashReportFields.h"
+#import "Private.h"
 
```

Comment:
Can/should these header changes be reverted now?

Reply:
Removed unnecessary header